lib/WP_Auth0_LoginManager.php

```diff
@@ -265,11 +265,11 @@ public function redirect_login() {
 
       $userinfo = json_decode( $response['body'] );
       if ( $this->login_user( $userinfo, $data->id_token, $data->access_token ) ) {
-        if ( null !== $stateFromGet && isset( $stateFromGet->interim ) && $stateFromGet->interim ) {
+        if ( ! empty( $stateFromGet->interim ) ) {
           include WPA0_PLUGIN_DIR . 'templates/login-interim.php';
           exit();
         } else {
-          if ( null !== $stateFromGet && isset( $stateFromGet->redirect_to ) ) {
+          if ( ! empty( $stateFromGet->redirect_to ) && wp_login_url() !== $stateFromGet->redirect_to ) {
             $redirectURL = $stateFromGet->redirect_to;
           } else {
             $redirectURL = $this->a0_options->get( 'default_login_redirection' );
```

Comment:
Is the next line equal? (without the use of `get`)

```php
$redirectURL = $this->a0_options->default_login_redirection;
```

Reply:
@lbalmaceda - I'm not sure what you're asking ...

Reply:
I'm asking if 

```php
$redirectURL = $this->a0_options->get( 'default_login_redirection' );
```

is the same as saying

```php
$redirectURL = $this->a0_options->default_login_redirection;
```

Reply:
No magic getters, no